core-web/libs/portlets/edit-ema/portlet/src/lib/store/features/editor/toolbar/withUVEToolbar.ts

```diff
@@ -221,6 +222,19 @@ export function withUVEToolbar() {
                     }
                 });
             },
+            setSEO: (socialMedia: string | null) => {
+                patchState(store, {
+                    device: null,
+                    orientation: null,
+                    socialMedia,
+                    viewParams: {
+                        ...store.viewParams(),
+                        device: null,
```

Comment:
we need to set the orientation to null

Reply:
PR is not ready yet, thanks for the feedback!